rust/forevervm/src/util.rs

```diff
@@ -40,3 +41,18 @@ impl From<Duration> for ApproximateDuration {
         Self::Seconds(duration.num_seconds())
     }
 }
+
+pub fn validate_email(email: &str) -> bool {
+    // from: https://stackoverflow.com/questions/201323/how-can-i-validate-an-email-address-using-a-regular-expression
+    let email_regex = Regex::new(r#"(?:[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*|"(?:[\x01-\x08\x0b\x0c\x0e-\x1f\x21\x23-\x5b\x5d-\x7f]|\\[\x01-\x09\x0b\x0c\x0e-\x7f])*")@(?:(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?|\[(?:(?:(2(5[0-5]|[0-4][0-9])|1[0-9][0-9]|[1-9]?[0-9]))\.){3}(?:(2(5[0-5]|[0-4][0-9])|1[0-9][0-9]|[1-9]?[0-9])|[a-z0-9-]*[a-z0-9]:(?:[\x01-\x08\x0b\x0c\x0e-\x1f\x21-\x5a\x53-\x7f]|\\[\x01-\x09\x0b\x0c\x0e-\x7f])+)\])"#).unwrap();
+    email_regex.is_match(email)
+}
+
+pub fn validate_account_name(account_name: &str) -> bool {
```

Comment:
We could move this (and the email validation) function to the `forevervm_sdk` if we want. That way we could pull it in both here and on the server since we'll have to do the same validation checks there anyway. Thoughts?

Reply:
Yeah, I like that